modules/ROOT/pages/ibm/ibm-mq-xml-ref.adoc

```diff
@@ -1,11 +1,11 @@
-= IBM MQ Connector Reference
+= IBM MQ Connector Reference - Mule 4
```

Comment:
General comments: There are sections where some descriptions have periods and some don't. I don't know if you care.

There are typos in text that I can't get to. I assume it is being automatically brought in and can't be edited. One of the places is with the MQ Ciphere field.